tshirts/TShirtsSizeTest.java

```diff
@@ -0,0 +1,14 @@
+
+
+public class TShirtsSizeTest {
+
+
+    public static void main(String[] args) {
+
+        assert(TShirts.size(37) == "S");
+        assert(TShirts.size(40) == "M");
+        assert(TShirts.size(43) == "L");
+        assert(TShirts.size(38) == "M"):"Invalid Size For Medium";
```

Comment:
make an assumption of what range size 38 belong to and assert the same.
for ex. you may consider size 38 as small , so add an assert for size 38 == small.

Add asserts for all boundary conditions including 42